charts/kueue/templates/kueueviz/frontend-deployment.yaml

```
@@ -22,7 +29,7 @@ spec:
             - containerPort: 8080
           env:
             - name: REACT_APP_WEBSOCKET_URL
-              value: "wss://backend.kueueviz.local"
+              value: wss://{{ .Values.kueueViz.backend.ingress.host | default "backend.kueueviz.local" }}
```

```suggestion
              value: 'wss://{{ .Values.kueueViz.backend.ingress.host | default "backend.kueueviz.local" }}'
```

Do we need single quotes like we have in other places?